web-components/src/components/tabs/TabPanel.ts

```diff
@@ -24,7 +24,6 @@ export namespace TabPanel {
     protected firstUpdated(changedProperties: PropertyValues) {
       super.firstUpdated(changedProperties);
       this.setAttribute("role", "tabpanel");
```

Comment:
Removing this is where I wonder about the A11y impact. A screen reader needs this to verbalize that a user has navigated to the Tab Content. The CSS is supposed to show the gighlight when navigated to with Key nav, but NOT show if a user clicks on inner elements with the mouse. If that is how it currently works, then we should keep it.

Reply:
Sure will be reverting this particular changes back